### DISCUSSION TITLE
Start public server with no password

### DISCUSSION THREAD

**Discussion by Phoenix-Game** [mbround18/valheim-docker#593] [Q&A]
Hi, I'd like to start the server without password and with public 1 cause I need to be able to add server in monitoring using port 2457
Is there any ideas how is that possible to do?

**Comment by mbround18** [comment]
@Phoenix-Game this is a restriction of Valheim itself, you need to use a plugin like ValheimPlus to achieve this

**Reply by Phoenix-Game** [reply]
Will it work if I'll use BetterServerConfig for that?

**Reply by mbround18** [reply]
@Phoenix-Game i am unsure! :/ i havent used that mod, would you mind giving it a test and seeing if that would work?

**Reply by Phoenix-Game** [reply]
I'll answer you if it works or not

**Reply by spannerman79** [reply]
ValheimPlus (known as V+) is the most common way to do this.  There are some other mod alternatives out there such as https://www.nexusmods.com/valheim/mods/578/ but the **only** way to do this is modded.

**Reply by Phoenix-Game** [reply]
I used BetterServerConfig but it doesn't work, in fact mod works well and server should start but I get the message that the server can't be public without password. So it doesn't start, but I can start this server with the same mods but not through docker. It seems to be that the problem is not with valheim server but with the container which don't even try to start the server.

**Reply by Phoenix-Game** [reply]
> ValheimPlus (known as V+) is the most common way to do this. There are some other mod alternatives out there such as https://www.nexusmods.com/valheim/mods/578/ but the **only** way to do this is modded.

I'll try to use this mod insteed of BetterServerConfig but for sure I'm not gonna use V+ it's an awful mod.

**Reply by spannerman79** [reply]
>in fact mod works well and server should start but I get the message that the server can't be public without password

That was rasied by me and fixed some time ago, see https://github.com/mbround18/valheim-docker/issues/294#issuecomment-807996856
The conditions needed are `TYPE` not to be Vanilla and `PUBLIC` is set to `0`.  The server **cannot** be public without a password.  This is a hard requirement by Iron Gate.

>I'll try to use this mod insteed of BetterServerConfig

Ahh, I had a read through many of the BetterServerConfig discussions.  It seems this mod is meant for those that have no fucking idea how to run a server.  Let alone run docker instances of _any_ server be it Valheim or otherwise.  I'll put this down as incorrect use case of mod as you do know how to use docker at least compared to some of those other discussions over there.

>but for sure I'm not gonna use V+ it's an awful mod.

How so?  If there is a mod compat issue create an issue over at the V+ repo.  If its a mod issue - the same thing - create an issue.

Every section of V+ can be toggled on or off, _many_ sections are toggled off by default.  So how is it "an awful mod" with actual evidence instead of conjecture.  

Because if conjecture starts to be allowed in discussions there are **many** discussions here that I've held back on from my bias point of view on other peoples crap choices.
